### DISCUSSION TITLE
Almost no punctuation and letter capitalization in transcript

### DISCUSSION THREAD

**Discussion by 7k50** [openai/whisper#290] [Q&A] [answered]
Most of my transcriptions using the Medium English model have been largely correct and impressive in terms punctuation and letter capitalization. However, on one occasion there was very little punctuation and letter capitalization done. 

It seems like the lack of proper punctuation is what causes there to be little letter capitalization. The first word in the transcript is "hello" in lower case, and it transcript continues to use lower case except for names/etc, and essentially no punctuation. 

I do not believe the problem is bad audio quality; the audio file is perfectly intelligible. 

It almost seems more like a "stylistic" decision by Whisper. The audio is three people talking about observations the world, with a bit of banter and laughs in-between, perhaps making it seem "casual". To be clear, I have no idea whether Whisper actually employs any kind of "stylistics" in how it transcribes – but it does sometime seem to me like like it does, because different audio sources (like podcasts) do have different row lengths, perhaps relating to the pace and content of the audio.

Is there any way to "force" Whisper to employ more thorough sentence parsing and letter capitalization? Is what I encountered a "feature" or more likely some sort of bug/problem?

**Comment by jianfch** [comment, marked as answer]
check #194
